### PR TITLE
add comments and initilze project

### PR DESCRIPTION
**Pull request by yonibc**

### CODE REVIEW

File: controllers/CommentController.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
+const Comment = require("../models/Comment");
+const { ObjectId } = require("mongoose");
+
+const createComment = async (req, res) => {
+    console.log(req.body);
+    try {
+      const comment = await Comment.create(req.body);
+      res.status(201).send(comment);
```
Comment (orshohat1) [inline controllers/CommentController.js]: change to 200

File: controllers/CommentController.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    try {
+      const comment = await Comment.findById(id);
+      res.status(200).send({comment});
+    } catch (err) {
+      res.status(400).send(err.message);
+    }
+};
+
+const getAllPostComments = async (req, res) => {
+    const { postId } = req.body;
```
Comment (orshohat1) [inline controllers/CommentController.js]: postId should be passed from the params and not body

File: controllers/CommentController.js [hunk 3]
```diff
@@ -0,0 +1,66 @@
+const Comment = require("../models/Comment");
+const { ObjectId } = require("mongoose");
+
+const createComment = async (req, res) => {
+    console.log(req.body);
+    try {
+      const comment = await Comment.create(req.body);
```
Comment (orshohat1) [inline controllers/CommentController.js]: no validation if the comment's content is not empty

File: controllers/CommentController.js [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+        });
+        res.status(200).send({comments});
+    } catch (err) {
+        res.status(400).send(err.message);
+    }
+    };
+
+const editComment = async (req, res) => {
+    const { id } = req.params;
+    const { content } = req.body;
```
Comment (orshohat1) [inline controllers/CommentController.js:42]: add validation to check if the comment content is not empty

File: controllers/CommentController.js (hunk 1 above)
Comment (yonibc) [reply controllers/CommentController.js]: `201` indicates that operation was successful and resource(s) has been created
`200` indicates that operation was successful only
so I think `201` is used correctly

File: controllers/CommentController.js (hunk 2 above)
Comment (yonibc) [reply controllers/CommentController.js]: how do you expect the url to look like?

Comment (orshohat1) [reply controllers/CommentController.js]: `router.get("/postComments/:postId", commentController.getAllPostComments);
`
for example

File: controllers/CommentController.js (hunk 3 above)
Comment (yonibc) [reply controllers/CommentController.js]: fixed

File: controllers/CommentController.js (hunk 4 above)
Comment (yonibc) [reply controllers/CommentController.js:42]: fixed

File: controllers/CommentController.js (hunk 2 above)
Comment (yonibc) [reply controllers/CommentController.js]: done.

**Pull request merged**